### ISSUE TITLE
Default loglevel info

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/stryker-net#159]
I was just thinking about how we could clean the repo a bit and noticed the status reporter might not be needed anymore.

## Default behaviour

```
Analyzing project
Building project
Starting initial testrun
Project OK
Generating mutants

1 mutants have been created. Each mutant will now be tested, this could take a while.
```
## --log-console info behaviour
```
Analyzing project
[11:47:42 INF] Using C:\Dev\ExampleProjects\Example2.1\ConsoleApp1\XUnitTestProject1\XUnitTestProject1.csproj as project file
Building project
[11:47:42 INF] Starting initial build
[11:47:43 INF] Initial build successful
[11:47:44 INF] Using testrunner DotnetTestRunner
Starting initial testrun
[11:47:44 INF] Initial testrun started
[11:47:47 INF] Initial testrun successful in 2196 ms
[11:47:47 INF] Using 33294 ms as testrun timeout
Project OK
Generating mutants
[11:47:47 INF] 1 mutants created
[11:47:49 INF] 1 mutants ready for test

1 mutants have been created. Each mutant will now be tested, this could take a while.
```
In the second example we see a lot of double information. By making the default loglevel "information" we could clean the repo quite a bit. This would mean:
* The [status reporter](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/Reporters/ConsoleStatusReporter.cs) could be removed.
* Some methods on the [reporter interface](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/Reporters/IReporter.cs) can be removed, which makes the other reporters a lot cleaner.
* The "ReportOnly" reporter setting will not be needed anymore, since users can just set the loglevel to warning or error to accomplish that.

@Mobrockers @simondel @yaelkeemink what do you think about this? Would this be a good issue for hacktoberfest?

**Comment by Mobrockers** [comment]
I agree. We had decided the reportonly reporter would be removed in the future anyway.

**Comment by lkts** [comment]
Let me take a look at this.
